novawallet/Modules/PayCard/PayCardInteractor.swift

```diff
@@ -61,6 +73,19 @@ extension PayCardInteractor: PayCardInteractorInputProtocol {
         }
     }
 
+    func processSuccessTopup() {
+        guard settingsManager.novaCardStatus != .created else {
+            return
+        }
+
+        let pendingStatus = PayCardStatus.pending
+
+        settingsManager.novaCardOpenTimeStamp = UInt64(Date().timeIntervalSince1970)
+        settingsManager.novaCardStatus = pendingStatus
+
+        presenter?.didReceiveCardStatus(pendingStatus)
```

Comment:
We don't send timestamp here to the presenter after topup and that is the reason why the pending bottom sheet is not shown